release.gradle

```diff
@@ -5,15 +5,15 @@ def isReleaseBuild() {
     return VERSION_NAME.contains("SNAPSHOT") == false
 }
 
-def getRepositoryUrl() {
+def getMavenRepositoryUrl() {
```

Comment:
Ack'd, I wonder if Google added the support upstream that we had to manually hack in.  I am a little worried that `./gradlew uploadArchives` may have been broken by this change as well but wasn't invoked during your tests.  I guess we'll just have to take that pain when it comes to release time...